### ISSUE TITLE
GEMMA

### ISSUE THREAD

**Issue by baizm** [ubccr/software-layer#340]
GEMMA
version 0.98.5 2022 (latest)
https://github.com/genetics-statistics/GEMMA

**Comment by tonykew** [comment]
GEMMA version 0.98.5 has been built, and is going through internal testing.
I will update this request once Nextflow has been published.

**Comment by tonykew** [comment]
GEMMA version 0.98.5 has been published to the ccrsoft/2023.01 software release


```
login1$ module spider gemma
----------------------------------------------------------------------------
  gemma: gemma/0.98.5
----------------------------------------------------------------------------
    Description:
      Genome-wide Efficient Mixed Model Association

    Properties:
      Bioinformatic libraries/apps

    You will need to load all module(s) on any one of the lines below before the "gemma/0.98.5" module is available to load.

      gcc/11.2.0  openmpi/4.1.1
[...]
login1$ module load gcc/11.2.0 openmpi/4.1.1 gemma/0.98.5
login1$
```

This includes "Gemma" and a quick test program "unittests-gemma"

```
login1$ ls ${EBROOTGEMMA}/bin
gemma  unittests-gemma
login1$ 
```

The test prog runs in a few seconds

```
login1$ unittests-gemma 
[...]
All tests passed (23 assertions in 4 test cases)

login1$ 
```

"gemma" itself:


```
login1$ gemma
GEMMA 0.98.5 (2021-08-25) by Xiang Zhou, Pjotr Prins and team (C) 2012-2021

 type ./gemma -h [num] for detailed help
 options: 
  1: quick guide
  2: file I/O related
  3: SNP QC
  4: calculate relatedness matrix
  5: perform eigen decomposition
  6: perform variance component estimation
  7: fit a linear model
  8: fit a linear mixed model
  9: fit a multivariate linear mixed model
 10: fit a Bayesian sparse linear mixed model
 11: obtain predicted values
 12: calculate snp variance covariance
 13: note
 14: debug options

The GEMMA software is distributed under the GNU General Public v3
   -license    show license information
   see also http://www.xzlab.org/software.html, https://github.com/genetics-statistics
login1$ 
```
